python/lsst/ts/eas/eas_csc.py

```diff
@@ -73,6 +74,7 @@ def __init__(
             override=override,
         )
         self.eas = None
+        self.m1m3_thermal_system: ControlLoopManager | None = None
```

Comment:
you probably don't need to have this as `None`. Just initialize the class here already, you have everything you need.

Reply:
I wound up taking away the class entirely. See what you think.